examples/petsc/bps.c

```diff
@@ -408,15 +410,24 @@ int main(int argc, char **argv) {
   ierr = VecSetUp(X); CHKERRQ(ierr);
 
   if (!test_mode) {
+      ierr = PetscPrintf(comm,
+                         "\nCEED Benchmark Problem %d:\n"
+                         "  backend        : %s\n"
+                         "  p              : %d\n"
+                         "  q              : %d\n",
```

Comment:
Longer (unambiguous) names for these would be better.

Are there any parsing changes that should go into the benchmarks/ scripts?

Reply:
I can expand the names.

There are parsing changes that would be required, but I wanted to get opinions on the sample output first.